docs/guide/workspace.md

```diff
@@ -10,11 +10,11 @@ title: Workspace | Guide
 
 :::
 
-Vitest provides built-in support for monorepos through a workspace configuration file. You can create a workspace to define your project's setups.
+Vitest provides a way to define multiple project configurations within a single Vitest process. This feature is particularly useful for monorepo setups but can also be used to run tests with different configurations, such as `resolve.alias`, `plugins`, or `test.browser` and more.
 
 ## Defining a Workspace
 
-A workspace must include a `vitest.workspace` or `vitest.projects` file in its root directory (located in the same folder as your root configuration file, if applicable). Vitest supports `ts`, `js`, and `json` extensions for this file.
+A workspace must include a `vitest.workspace` or `vitest.projects` file in its root directory (located in the same folder as your root configuration file or working directory if it doesn't exist). Vitest supports `ts`, `js`, and `json` extensions for this file.
```

Comment:
Something that's not clear to me here: is `vitest.projects` a synonym for `vitest.workspace` with the same file format or does it have a separate accepted syntax? Is there a reason two names are used and is one less preferred than the other?

Reply:
They are aliases of each other; there is no special handling. The `projects` is supported because this is what it is called in Jest